### ISSUE TITLE
Similar Interests: no results for matching profiles

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#6309]
### Expected behavior
When I search for profiles with similar interests I should get some results.

### Actual behavior
When I search for profiles with similar interests I get no results. Zero.

### Steps to reproduce the problem
Go to .../contact by clicking on the contacts icon. Click on the link for "Similar Interests". You will be redirected to .../match but there are no results.

**From the log:**
````
PHP Notice:  Trying to get property 'results' of non-object in .../mod/match.php on line 74
PHP Warning:  count(): Parameter must be an array or an object that implements Countable in .../mod/match.php on line 74
````

### Friendica version you encountered the problem
2018.12-rc

### Friendica source (git, zip)
git

### PHP version
7.2.10

### SQL version
10.1.34 MariaDB

**Comment by MrPetovan** [comment]
Did you set your `system.directory` to an instance of `friendica/friendica-directory`? If yes, the `/msearch` is missing from the new directory version, which would explain the unexpected `json_decode` result. See progress here: https://github.com/friendica/friendica-directory/issues/20

**Comment by AlfredSK** [comment]
Yes. I am running a directory at https://dir.libranet.de (v.2.1.1) which I use as directory for my nodes.
But I also tried to set https://dir.friendica.social as directory in my admin panel. The 'match' didn't work either.

**Comment by MrPetovan** [comment]
I'm... actually not surprised.

**Comment by AlfredSK** [comment]
Yeah! :tada: 

![bildschirmfoto von 2018-12-24 23-56-43](https://user-images.githubusercontent.com/3656054/50407473-03d2fc80-07d8-11e9-8eff-e82cb2923b64.png)

**Comment by AlfredSK** [comment]
Yes, you're right. But I wanted to show my Gimp skills. :-)

**Comment by MrPetovan** [comment]
IMPRESSIVE!
